pages/docs/castor/rao-steps.md

```diff
@@ -46,6 +46,14 @@ An automaton perimeter can be defined for each defined contingency. Each automat
 Overloads potentially reached on this perimeter can possibly be solved with automatons (ARA). 
 Optimal preventive actions selected by the preventive RAO are applied in this perimeter.  
 
+The automaton perimeter is not designed to optimise automatic remedial actions, but to simulate their use. This simulation is carried out in two stages:
+
+1. First, all automatic network remedial actions (topological actions, injection set-points, PST set-points, switch paurs) are applied directly on the newtork without any need for optimisation (since there is no set-point to compute)
```

Comment:
Pourrais tu étoffer au début en justifiant la simulation par la modélisation du comportement réel des automates ? 
coquilles : switch pairs, network
je trouve ça ambigu le"without any need for optimisation" du coup vu qu'on dit bien qu'on n'optimise pas. 
Pour la deuxième partie, pourrais tu étoffer stp notamment en expliquant qu'on calcule le setpoint optimal à partir des sensis (voire mettre la formule)
coquille sur iterative